examples/manipulating_visualizing/plot_visualize_megatrawls_netmats.py

```diff
@@ -0,0 +1,49 @@
+"""
+Visualizing Megatrawls Network Matrices from Human Connectome Project
+=====================================================================
+
+This example shows how to visualize network matrices fetched from
+HCP beta-release of the Functional Connectivity Megatrawl
+
+For this, we need a fetcher named as :func:`nilearn.datasets.fetch_megatrawls_netmats` in
+nilearn.datasets
+
+Please see related documentation for more details.
```

Comment:
Shorten the two sentences:

See :func:`nilearn.datasets.fetch_megatrawls_netmats` documentation for more details.